less/current/modules/tables.less

```diff
@@ -88,3 +88,50 @@ table.submitted-table > tbody > tr > td:nth-child(9) > a:first-child {
 .offset3 {
   margin-left: 260px;
 }
+.guide-table {
```

Comment:
this replaces embedded styles at the top of pages as well as inline table styles